src/io/flutter/view/InspectorPerfTab.java

```diff
@@ -27,14 +33,14 @@
     add(Box.createVerticalStrut(6));
 
     Box labelBox = Box.createHorizontalBox();
-    labelBox.add(new JLabel("Running in " + app.getLaunchMode() + " mode"));
+    labelBox.add(new JBLabel("Running in " + app.getLaunchMode() + " mode"));
     labelBox.add(Box.createHorizontalGlue());
     labelBox.setBorder(JBUI.Borders.empty(3, 10));
     add(labelBox);
 
     if (app.getLaunchMode() == FlutterLaunchMode.DEBUG) {
       labelBox = Box.createHorizontalBox();
-      labelBox.add(new JLabel("Note: for best results, re-run in profile mode"));
+      labelBox.add(new JBLabel("<html><body><p style='color:red'>WARNING: for best results, re-run in profile mode</p></body></html>"));
```

Comment:
Unrelated one line change: show the warning in red so users realize they should use debug mode. I can revert this if desired.

Reply:
👍 looks good